src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncMainTests.cs

```diff
@@ -880,7 +976,7 @@ class A
         await Task.Factory.StartNew(() => { });
     }
 }";
-            CreateCompilationWithMscorlib45(source, options: TestOptions.ReleaseExe).VerifyDiagnostics(
+            CreateCompilationWithMscorlib45(source, options: TestOptions.ReleaseExe, parseOptions: TestOptions.Regular.WithLanguageVersion(LanguageVersion.CSharp7)).VerifyDiagnostics(
```

Comment:
>, parseOptions: TestOptions.Regular.WithLanguageVersion(LanguageVersion.CSharp7) [](start = 83, length = 80)

If we don't expect the behavior to be version specific, we probably shouldn't use specific version for this test. Perhaps we should test it against default version and the latest version, latest will give us coverage for 7.1.